pyproject.toml

```diff
@@ -3,7 +3,8 @@ requires = [
   "setuptools",
   "wheel",
   "cython>=0.29.32",
-  "oldest-supported-numpy",
+  "numpy==2.0.0; python_version >= '3.9'",
+  "oldest-supported-numpy; python_version < '3.9'"
```

Comment:
I don't think it's recommended to use this anymore. I can find the link when I'm at home

Reply:
https://numpy.org/doc/stable/dev/depending_on_numpy.html#numpy-2-0-specific-advice I think you want to ensure that you're using numpy 2 for building the wheels

Reply:
🙏

Reply:
looking at other project they are doing this because numpy 2 is not provided for python 3.8 (e.g rasterio, shapely) 

ref https://numpy.org/doc/stable/dev/depending_on_numpy.html#build-time-dependency